src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+/**
+ * Class for Ui components of Duke.
+ */
+public class Ui {
+
+    public static String divider() {
+        return "--------------------";
+    }
+
+    protected static void setup() {
+//        System.out.println("Working Directory = " + System.getProperty("user.dir"));
+        System.out.println("Hello from " + "DUKE");
+        System.out.println("What can I do for you?");
+        System.out.println(divider());
+    }
+
+    protected static String checkbox(Boolean mark) {

```

Comment:
I think this method should be renamed into a verb to follow the coding standards, perhaps something like getCheckBox()?